File: ChurchLib/ChurchLib/Models/LibraryModels/Reader.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace ChurchLib.Models.LibraryModels
{
    public class Reader
    {
        public int ReaderID { get; set; }
        [StringLength(20)]
        public string firstName { get; set; }
        [StringLength(20)]
        public string lastName { get; set; }
        [Required]
        public string email { get; set; }
        public string phone { get; set; }

        public virtual ICollection<BorrowBook> BorrowedBooks { get; set; }
    }
}

File: src/pages/reader-list/reader.ts
import { BorrowedBook } from "../book-list/borrowed-book";

export class Reader {
    public constructor(
        public ReaderID: number,
        public firstName: string,
        public lastName: string,
        public email: string,
        public phone: string,
        public BorrowedBooks: BorrowedBook[]
    ){}
}